mlflow/pytorch/pytorch_autolog.py

```diff
@@ -90,17 +91,11 @@ def on_train_end(self, trainer, pl_module):
         Logs the model checkpoint into mlflow - models folder on the training end
         """
 
-        tempdir = tempfile.mkdtemp()
-        if os.path.exists(tempdir):
-            shutil.rmtree(tempdir)
-
-        try:
-            mlflow.pytorch.save_model(trainer.model, path=tempdir)
-            trainer.logger.experiment.log_artifact(
-                trainer.logger.run_id, local_path=tempdir, artifact_path="model"
-            )
-        finally:
-            shutil.rmtree(tempdir)
+        mlflow.set_tracking_uri(trainer.logger._tracking_uri )
```

Comment:
Pytorch Lightning's MLFlow Logger and mllfow.pytorch are two different libraries. Adding the explicit start and end run with the same run id of MLFlowLogger to log the data into the same experiment.